src/Ticket.php

```diff
@@ -4980,6 +4980,12 @@ public static function showCentralList($start, $status = "process", bool $showgr
                         $options['criteria'][1]['value']      = '^';
                         $options['criteria'][1]['link']       = 'AND';
 
+                        $duration = Entity::getUsedConfig('inquest_duration', $_SESSION['glpiactive_entity']);
+                        if ($duration > 0) {
+                            $options['criteria'][1]['searchtype'] = 'morethan';
+                            $options['criteria'][1]['value']      = '-' . $duration . 'DAY';
+                        }
```

Comment:
The problem is that this configuration may differ from one entity to another, and this link may display satisfactions from multiple entities at once. Depending on the entities configuration, the current proposal could filter surveys from a child entity that are still valid.
If we had the computed values for each entity, building a new search option to apply the correct filter would be easy, but we cannot change this in GLPI 10.0.
I do not have any other viable solution to propose.

For information, there is also a fixed for the list displayed in the central page in #17249 (not reviewed/validated yet).